github/GithubObject.py

```diff
@@ -87,6 +88,26 @@ def remove_unset_items(data: Dict[str, Any]) -> Dict[str, Any]:
         }
 
 
+def is_defined(v: Union[T, _NotSetType]) -> TypeGuard[T]:
+    return not isinstance(v, _NotSetType)
+
+
+def is_undefined(v: Any) -> TypeGuard[_NotSetType]:
+    return isinstance(v, _NotSetType)
+
+
+def is_optional(v, type: Type[T]) -> TypeGuard[List[T]]:
```

Comment:
Why is this `TypeGuard[List[T]]` vs `TypeGuard[Optional[T]]`?

Reply:
Oh, this is wrong... it's `TypeGuard[Opt[T]]`